src/chat.cpp

```diff
@@ -50,6 +50,8 @@ ChatBuffer::ChatBuffer(u32 scrollback):
 
 void ChatBuffer::addLine(const std::wstring &name, const std::wstring &text)
 {
+	m_lines_modified = true;
```

Comment:
This should be done where the lines are actually modified -> `formatChatLine` and `deleteOldest` so that `reformat()` properly updates this boolean as well (indirectly).

Reply:
`m_unformatted` is not modified by `reformat()`, see comments at declaration of `m_lines_modified`.